cgt/api.py

```diff
@@ -608,7 +608,7 @@ def stack(tensors, axis=0):
     along all axes
 
```

Comment:
You may also want to document `axes :` while you're at it. Maybe just copy the Theano docstring, that one explains it quite well!